### DISCUSSION TITLE
Question: seam_gap and bridges

### DISCUSSION THREAD

**Discussion by eaannist** [supermerill/SuperSlicer#2208] [Q&A]
I really like the seam_gap parameter, it's very useful for avoiding overextrusion at junction points. My question is: does it apply also for the end-junctions between bridges lines and perimeter? 
I did all the calibration needed for bridjes, over_bridge_flow_ratio=100%  worked awesome for me. After setting the seam_gap parameter to 1mm, i got very clean junction points on loops but when bridging several lines miss connecting with perimeter.

If the answer is yes, which parameters can help me for getting better bridges? Do i have to adjust over_bridje_flow_ratio again? Or a new boolean condition will be needed to disable seam_gap when bridging?

I hope my question wasn't stupid and great thanks to all the developers for the magnificent job.

**Comment by supermerill** [comment]
seam_gap is used when there is a loop and it's cut.
bridges aren't loops but a kind of rectilinear infill.
Now, there is the overhangs that can be a kind of bridge. The slicer try to not cut the loop on overhangs for obvious reasons. But maybe the seam_gap may extends the cut over it. Is it your problem?

**Comment by tocguy** [comment]
"over-bridge flow ratio" cannot be located in the settings. It appears that the calibration procedures and the setting's name are no longer in sync. Closest match I found is located in Print Settings -> Width & Flow is "Above the bridges". It is challenging to climb the learning curve of subtle inter-relationships of all these settings thus it is important that the calibration procedures are clear and practical. For the most part they are but it is apparent that the level of explanatory detail in these calibration procedures falls off rapidly beginning with Bridge-Flow and that the naming discontinuity explained above indicates that time was a constrained resource when assembling these calibration procedures.

Hope the above info is received as constructive information. Other than that, I find Klipper to be head and shoulders above the default firmware provided with my machine. In short, Klipper represents a 250% improvement in both speed and quality which is interesting given that those characteristics are usually at odds with one another.
